xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-pageobjects/src/main/java/org/xwiki/ckeditor/test/po/RichTextAreaElement.java

```diff
@@ -155,4 +156,18 @@ private WebElement getActiveElement()
         getDriver().switchTo().frame(this.iframe);
         return getDriver().switchTo().activeElement();
     }
+    
+    /**
+     * Waits until a macro appears in the document.
+     */
+    public void waitUntilMacroAppears()
```

Comment:
I find this too specific. I'd rather have a more generic "wait until the content is re-loaded".

Reply:
I'm not sure how to tell that the content has been reloaded. The content refreshes but I believe the ckeditor widget take some time to appear?